dash-renderer/src/reducers/callbacks.ts

```diff
@@ -0,0 +1,147 @@
+import {
+    concat,
+    difference,
+    reduce
+} from 'ramda';
+
+import {
+    ICallback,
+    IExecutedCallback,
+    IExecutingCallback,
+    IStoredCallback
+} from '../types/callbacks';
+
+export enum CallbackActionType {
+    AddApplied = 'Callbacks.AddApplied',
```

Comment:
is `applied` in use?

Reply:
🔪  https://github.com/plotly/dash/pull/1254/commits/ae38bf8d2e7cb125c3fb4a0944b93c4e121787eb